examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>by exploding our n observations -&gt; by reparametrizing our ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I thought "exploding" was more evocative and pairs well with "collapsing". How do you feel about:</p><p> </p><blockquote>Because of a couple of identities of the multinomial distribution, we can reparameterize this model in a number of ways—we could explode...</blockquote>

Reply:
Sounded a bit strange to me. Fun verb for sure!